ds_judgements_public_ui/templates/includes/results_search_component.html

```diff
@@ -0,0 +1,11 @@
+<div class="results-search-component">
+  <form action="{% url 'results' %}" id="analytics-results-filters">
+
+    <div class="results-search-component__container">
+      {% include 'includes/results_search_field.html' %}
+      {% include 'includes/results_applied_filters.html' %}
+    </div>
+
+    {% include 'includes/results_filters.html' %}
+  </form>
+</div>
```

Comment:
Hi @LornaHarwood. We have `crispy-forms` installed on this project, which should allow us to avoid a lot of this custom template work. We should be aiming to do as much of this in python as possible: 
https://django-crispy-forms.readthedocs.io/en/latest/layouts.html

Reply:
On second thoughts, maybe there's no way to avoid dedicated templates for the level of customisation being made from a UI perspective - if it's only applicable to the search form, and the fields/filters are pretty stable, shifting over to `crispy-forms` might not be worth it.

Reply:
I don't have a particularly strong view on whether the custom template approach we've used in Flask could be achieved in a more streamlined way using `crispy-forms` in Django but I'd probably opt for the approach that will be most time efficient and easiest for people to work with in the short term.

Reply:
We could take either approach, but let's treat it as a separate "swap to crispy forms" task later on, if we want to.